api/installation_service.go

```diff
@@ -52,3 +60,38 @@ func (is InstallationService) Export(outputFile string) error {
 
 	return nil
 }
+
+func (is InstallationService) Import(input ImportInstallationInput) error {
```

Comment:
Can't use diagnostic service for this -- we do not provide a username and password when we import the installation, so there's no way the diagnostic service can get an authenticated client even if the ops manager is configured. I would say we ditch that and just merge this PR as is. The 422 check is good enough.